manifests/istio/ingressgateway-acme-cert/base/certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  namespace: $(namespace)
```

Comment:
I don't think `$(namespace)` is needed, kustomize can handle it.

Reply:
what do you mean? how it can handle? if I just remove names from here where this certificate will be created?

Reply:
You don't need to have a variable $(namespace), customize can set the namespace for you

Reply:
to the one that specified in params?

Reply:
so if you specified `namespace` parameter in kfdef, you don't need to put it in params.env, and here and kustomization.yaml vars, and varsreference, because kfctl/kustomize can already support `namespace`, if you look into overlay of ingressgateway, you will see we set `namespace` in parameter in kfdef, and it will be passed by kfct/kustomize to overlays.